index.html

```diff
@@ -45,6 +45,342 @@ <h2 class="supporting">
                 </ul>
             </span>
         </section>
+        <section class="recent-works">
+            <span class="title-container">
+                <h2>My Recent Works</h2>
+                <hr />
+            </span>
+            <div class="frame-44">
+                <img src="./imgs/Img_Placeholder.png" alt="featured image">
+                <div class="right-block">
+                    <h2>
+                        Multi-Post Stories
+                    </h2>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                        the industry's standard
+                        dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="tech-skills">
+                        <li class="css">
+                            <span>css</span>
+                        </li>
+                        <li class="html">
+                            <span>html</span>
+                        </li>
+                        <li class="bootstrap">
+                            <span>bootstrap</span>
+                        </li>
+                        <li class="ruby">
+                            <span>Ruby</span>
+                        </li>
+                    </ul>
+                    <button class="see-project-btn">
+                        <span>See Project</span>
+                    </button>
```

Comment:
- [ ] Kindly add button interaction styles to all the buttons on the page as shown in Figma 💯 

![image](https://user-images.githubusercontent.com/50721479/213866357-a31c0d1f-2cd7-4327-a325-43c3feda4e87.png)

Reply:
Done